src/components/Match/matchColumns.jsx

```diff
@@ -657,7 +657,7 @@ export default (strings) => {
                 })
                 .map((purchase) => {
                   if (items[purchase.key] && (showConsumables || items[purchase.key].qual !== 'consumable')) {
-                    return inflictorWithValue(purchase.key, formatSeconds(purchase.time));
+                    return inflictorWithValue(purchase.key, formatSeconds(purchase.time), 'purchase', null, null, purchase.charges);
```

Comment:
is the new type 'purchase' needed here? We can just display charges if it's provided, right?